packages/suite/src/components/suite/layouts/SuiteLayout/utils.ts

```diff
@@ -1,9 +1,39 @@
+import { createDeviceInstance, selectDevice } from '@suite-common/wallet-core';
+import { useEvent } from 'react-use';
+import { goto } from 'src/actions/suite/routerActions';
 import { useCustomBackends } from 'src/hooks/settings/backends';
-import { useSelector } from 'src/hooks/suite';
+import { useDispatch, useSelector } from 'src/hooks/suite';
 
 export const useEnabledBackends = () => {
     const enabledNetworks = useSelector(state => state.wallet.settings.enabledNetworks);
     const customBackends = useCustomBackends();
 
     return customBackends.filter(backend => enabledNetworks.includes(backend.coin));
 };
+
+export const useAppShortcuts = () => {
```

Comment:
I would prefer one logical block in separate file. this utils file can contain anything. IMO we should avoid anything named `utils`.